lastuser_core/utils.py

```diff
@@ -41,6 +40,8 @@ def strip_phone(candidate):
 def valid_phone(candidate):
     return not PHONE_VALID_RE.search(candidate) is None
 
+def valid_username(candidate):
+    return not USERNAME_VALID_RE.search(candidate) is None
```

Comment:
Where was this introduced from? Also, extra blank line required above and below.